src/multiserialviewer/application/serialViewerController.py

```diff
@@ -16,14 +16,15 @@ def __init__(self, receiver: SerialDataReceiver, processor: SerialDataProcessor,
         self.view: SerialViewerWindow = view
 
         self.view.signal_closed.connect(self.terminate)
+        self.receiver.rawData.connect(self.processor.handleRawData)
```

Comment:
yes. I think its a good idea to "highlight" the signals somehow as the code deals a lot with them. The identifer looks a litte clunky with signal_* but only s_ is also not an option as it can be also a "slot" :D. What about sig_* ?

Reply:
3 characters less is not a big difference... But I don't like sig_ because its not a word you can read like normal text.

Reply:
yes that also correct. So lets use signal_*.